### DISCUSSION TITLE
how to do optimization with exception

### DISCUSSION THREAD

**Discussion by jjhesk** [webpack/webpack#14496] [General]
Followed by the previous [discussion](https://github.com/nuxt/nuxt.js/discussions/9957), I want to make an exception for the following class name "ABC" and how does it get it working? does it have any example?  https://github.com/nuxt/nuxt.js/discussions/9957 @simon04 @ooflorent

**Comment by alexander-akait** [comment]
Can you clarify and provide more information?
